src/components/settings/SettingsSSOTab/NewExistingSSOConfigs.jsx

```diff
@@ -0,0 +1,196 @@
+
+  useQuery({
+    queryKey: ['ssoOrchestratorConfigPoll'],
+    queryFn: async () => {
+      const res = await LmsApiService.listEnterpriseSsoOrchestrationRecords(enterpriseId);
+      const inProgress = res.data.filter(
+        config => (config.submitted_at && !config.configured_at) || (config.configured_at < config.submitted_at),
+      );
+      const untested = res.data.filter(config => !config.validated_at || config.validated_at < config.configured_at);
+
+      if (queryForConfiguredConfigs) {
+        if (inProgress.length === 0) {
+          setRefreshBool(!refreshBool);
+          setQueryForConfiguredConfigs(false);
+        }
+      }
+
+      if (queryForTestedConfigs) {
+        if (untested.length === 0) {
+          setRefreshBool(!refreshBool);
+          setQueryForTestedConfigs(false);
+        }
+      }
+
+      if (inProgress.length === 0 && untested.length === 0) {
+        queryClient.invalidateQueries({ queryKey: ['ssoOrchestratorConfigPoll'] });
+      }
+
+      return res.data;
+    },
+    // Refetch the data every second
+    refetchInterval: intervalMs,
+    enabled: queryForTestedConfigs || queryForConfiguredConfigs,
+    refetchOnWindowFocus: true,
+  });
+
+  useEffect(() => {
+    const [active, inactive] = _.partition(configs, config => config.active);
+    const inProgress = configs.filter(
+      config => (config.submitted_at && !config.configured_at) || (config.configured_at < config.submitted_at),
+    );
+    const untested = configs.filter(config => !config.validated_at);
+    const live = configs.filter(
+      config => (config.validated_at && config.active && config.validated_at > config.configured_at),
+    );
+    const notConfigured = configs.filter(config => !config.configured_at);
+
+    if (live.length >= 1) {
+      setLiveConfigs(live);
+      openAlerts();
+    }
+
+    setUntestedConfigs(untested);
+    if (untested.length >= 1) {
+      setQueryForTestedConfigs(true);
+      openAlerts();
+    }
+    setInProgressConfigs(inProgress);
+    if (inProgress.length >= 1) {
+      const beenConfigured = inProgress.filter(config => config.configured_at);
+      if (beenConfigured.length >= 1) {
+        setIntervalMs(UPDATED_CONFIG_POLLING_INTERVAL);
+      }
+      setQueryForConfiguredConfigs(true);
+      openAlerts();
+    }
+
+    if (notConfigured.length >= 1) {
+      setNotConfiguredConfigs(notConfigured);
+    }
+
+    setActiveConfigs(active);
+    setInactiveConfigs(inactive);
+    setLoading(false);
+  }, [configs, refreshBool, openAlerts]);
+
+  return (
+    <>
+      {!loading && (
+        <>
+          {showAlerts && (
+            <NewSSOConfigAlerts
+              liveConfigs={liveConfigs}
+              inProgressConfigs={inProgressConfigs}
+              untestedConfigs={untestedConfigs}
+              notConfigured={notConfiguredConfigs}
+              closeAlerts={closeAlerts}
+            />
+          )}
+          {activeConfigs.length > 0 && (
```

Comment:
nit: activeConfigs and inactiveConfig sections are identical aside from header text and config list, might be worth making a helper function for readability.